Mage/CommandCenter/TransportClient/CommandHTTPClient.cs

```diff
@@ -87,12 +87,12 @@ public class CommandHTTPClient : CommandTransportClient {
 			// Check if there was a transport error
 			if (requestError != null) {
 				string error = "network";
-				if (requestError is WebException) {
-					WebException webException = requestError as WebException;
-					HttpWebResponse webResponse = webException.Response as HttpWebResponse;
-					if (webResponse != null && webResponse.StatusCode == HttpStatusCode.ServiceUnavailable) {
-						error = "maintenance";
-					}
+
+				// On error
+				var wwwError = requestError as HTTPRequestException;
```

Comment:
According to the C# website: `The as operator is like a cast operation. However, if the conversion isn't possible, as returns null instead of raising an exception.`